src/Display/Progress/Progress.tsx

```diff
@@ -23,10 +23,12 @@ const Progress: React.FunctionComponent<Props> = (props) => {
 
   let normalizedPercentage;
   let normalizedSize;
-
-  if (percentage < 0 || percentage > 100) {
+  if (isNumber(percentage) && percentage < 0 || percentage > 100) {
     normalizedPercentage = percentage < 0 ? 0 : percentage > 100 ? 100 : percentage;
```

Comment:
@franciscrispin this expression, though correct, is hard to grok by another developer. Is there a more readable way of writing this?